### PR TITLE
Cockroach guts fix

### LINKED ISSUE #14359: cockroach guts are all snowflakey
why can they be dragged

### PR DESCRIPTION
**Pull request by swankcookie**
fixes #14359  Some dunce decided that he really liked typing "anchored = 1" over and over again, so there was no anchor assignment in /decal/cleanable. now there is, so now more /cleanable additions will need to be fixed.  I'll go back and get rid of the anchor tags individually, if that's what is needed.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
make it so you can lazarus the guts and get like 5 roaches from it

**Comment by swankcookie** [comment]
I can't see how that would end badly.

**Comment by Jordie0608** [comment]
Just change /obj/effect/decal/cleanable to be anchorced and remove it from all the children

**Comment by swankcookie** [comment]
I did. I just have to go through all the children and remove it.  I'll do it after work.

File: cfg/admin.txt [hunk 1]
```diff
@@ -12,4 +12,4 @@ cheridan
 giacomand
 rockdtben
 MrStonedOne
-Jordie0608
+Jordie0608
```
Comment (Aranclanos) [inline cfg/admin.txt]: stop touching this file

Comment (swankcookie) [inline cfg/admin.txt]: sry

**Comment by duncathan** [comment]
Yeah go through the children and remove the redundant definitions

**Pull request merged**